packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -151,6 +155,21 @@ export class AccountResolver {
   }
 
   @Features(AdminPanelFeature.AccountSearch)
+  @Mutation((returns) => Boolean)
```

Comment:
Just noticed that this method declares a return type of boolean, but nothing is returned. I suppose it'd be nice to return true if the db call appears to have succeeded.

Reply:
I think switching the return type void is acceptable. It looks like the result is never checked anyways.

Reply:
Just adding a note for posterity: I looked into this and apparently GQL frowns on returning void to the point where you need to define your own custom `Void` scalar (or import another 3rd party library to provide one for you.) Given this, I opted to modify the `create` method on `SecurityEvent` to return a bool, so that it more closely matched the other mutations.

Reply:
Alternatively return true would be fine.